app/assets/stylesheets/ama_layout/layout_components/siteheader.scss

```diff
@@ -13,6 +13,12 @@
   top: 0;
   width: 100%;
   z-index: 9999;
+
+  &--absolute-mobile{
```

Comment:
why are we setting it to be absolute on mobile? does it need to be a modifier, or can all instances of header be absolute on mobile?